### DISCUSSION TITLE
Spectral Unrecognized After Installation

### DISCUSSION THREAD

**Discussion by michaelcilibrasi** [stoplightio/spectral#2176] [Q&A]
I'm trying to install Spectral. It looks like everything installs correctly, but the spectral command is still unrecognized. 

Any ideas why?

![image](https://user-images.githubusercontent.com/83464206/172422880-706c4ef3-71b8-4977-8735-0c700986686d.png)

**Comment by philsturgeon** [comment]
This can depend on many things, like how you've got $PATH setup (or whatever the equivilent is in Windows land).

Are you having problems with other NPM modules installed by the CLI, or just Spectral? 

I use asdf and had to run `asdf reshim nodejs` after installing the latest spectral-cli.

**Comment by michaelcilibrasi** [comment]
Thanks for the reply.

I think I covered all of those for Windows. This worked successfully for my
personal Windows (as opposed to my work machine). The only differences now
is that I'm running A Windows emulator on the Mac, an application named
Parallels, and this is Windows 11. I wouldn't expect to see any forum notes
for Parallels, but there are no notes about Windows 11 either.

Do you think either of those might be the problem?
